workspaces/ui-v2/src/optic-components/pages/diffs/AddEndpointsPage.tsx

```diff
@@ -163,23 +167,40 @@ export function DocumentationRootPageWithPendingEndpoints(props: any) {
             {pendingEndpointsToRender.map(
               (endpoint: IPendingEndpoint, index: number) => {
                 return (
-                  <EndpointRow
+                  <ListItem
```

Comment:
Removed EndpointRow since it uses _mostly_ precomposed components (and that there were slight variations - e.g. changelog + different children). I think it's potentially easier / clearer to remove this layer of abstraction + it means I don't need to build a `DiffEndpointRow` and a `DocumentationEndpointRow`

Reply:
I think this makes sense, as well as the architecture choices motivating it.